explainer.md

```diff
@@ -772,4 +824,15 @@ partial dictionary WebGLContextAttributes {
 partial interface WebGLRenderingContextBase {
     Promise<void> setCompatibleVrDevice(VRDevice device);
 };
+
+//
+// RenderingContext
+//
+interface VRRenderingContext {
+  readonly attribute HTMLCanvasElement canvas;
+};
+
+dictionary VRRenderingContextSettings {
```

Comment:
Ah, I never mention that anywhere, do I? `getContext` allows a dictionary of context creation args for any context type and I was just trying to think of what properties might be applicable here. Honestly I'm not even sure this is necessary, though. I can drop if for now.